### ISSUE TITLE
Ethermine just closes when launching bat

### ISSUE THREAD

**Issue by Xeowolf** [ethereum-mining/ethminer#610]
Hello, I downloaded ethminer 0.13.0rc9 and then created a bat file. I ran into DLL issues. So I installed the microsoft visual basic 2015 and restarted my PC. Now when I launch the bat file it just closes instantly. I tried adding pause but it doesn't show anything useful.

Rig is using 1060s (4 of them)
Bat file reads:
setx GPU_FORCE_64BIT_PTR 0
setx GPU_MAX_HEAP_SIZE 100
setx GPU_USE_SYNC_OBJECTS 1
setx GPU_MAX_ALLOC_PERCENT 100
setx GPU_SINGLE_ALLOC_PERCENT 100

ethminer.exe -farm-recheck 2000 -U -s us1.ethermine.org:14444 -FS us2.ethermine.org:4444 -O ACTUALMININGADDRESSSNOTLISTED.HomeRig

What am I doing wrong?

**Comment by ZiDanRO** [comment]
Use only this in your bat and write your address correctly there:

`ethminer.exe --farm-recheck 10000 -U -S us1.ethermine.org:4444 -FS us2.ethermine.org:4444 -O ACTUALMININGADDRESSSNOTLISTED.HomeRig --cuda-parallel-hash 4  --cuda-grid-size 4096`

**Comment by Xeowolf** [comment]
Thanks for the reply. I realized I had invalid arguments when I added echo off and stepped through it. Oddly enough it wasn't reporting errors without echo off.  Out of curiosity what does the --cuda-parallel-hash 4 do for me? Does this help sync it to use the 4 gpus?
Note: I got it running using:
setx GPU_FORCE_64BIT_PTR 0
setx GPU_MAX_HEAP_SIZE 100
setx GPU_USE_SYNC_OBJECTS 1
setx GPU_MAX_ALLOC_PERCENT 100
setx GPU_SINGLE_ALLOC_PERCENT 100

ethminer.exe --farm-recheck 2000 -U -S us1.ethermine.org:14444 -FS us2.ethermine.org:4444 -O ACTUALMININGADDRESSSNOTLISTED.HomeRig
